pkg/client/client.go

```diff
@@ -0,0 +1,40 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/cloudwego/kitex/pkg/kerrors"
+	"github.com/kitex-contrib/kitexcall/pkg/config"
+	"github.com/kitex-contrib/kitexcall/pkg/errors"
+	"github.com/kitex-contrib/kitexcall/pkg/log"
+)
+
+type Client interface {
+	Init(conf *config.Config) error
+	Call() error
+	Output() error
+}
+
+func InvokeRPC(c Client, conf *config.Config) error {
+
+	if err := c.Init(conf); err != nil {
+		return errors.New(errors.ClientError, "client init failed: %v", err)
+	}
+
+	if err := c.Call(); err != nil {
+		// Handle Biz error
+		bizErr, isBizErr := kerrors.FromBizStatusError(err)
```

Comment:
BizStatusError is returned by the business code, so it should be in the output in a parsable format, for example, another key in the json output.

Reply:
done